benchmarks/cpp/nvfuser/softmax.cpp

```diff
@@ -182,21 +276,216 @@ static void Softmax_Dropout_Baseline(benchmark::State& benchmark_state) {
     benchmark_state.SetIterationTime(timer.elapsed() / 1000.0);
     cudaDeviceSynchronize();
   }
+
+  benchmark_state.SetBytesProcessed(
+      int64_t(benchmark_state.iterations()) * 6 * 256 * 12 * 100 *
```

Comment:
Maybe replace the hardcoded size `256 * 12 * 100 * benchmark_state.range(0)` with a constant or `attention_scores.numel()`

Reply:
Should this be `5 * input_numel * dataTypeSize(dtype) + input_numel * dataTypeSize(Bool)` where `input_numel = 256 * 12 * 100 * benchmark_state.range(0)`?